project_euler/problem_085/sol1.py

```diff
@@ -0,0 +1,46 @@
+"""
+Project Euler Problem 85: https://projecteuler.net/problem=85
+
+By counting carefully it can be seen that a rectangular grid measuring 3 by 2
+contains eighteen rectangles.
+￼
+Although there exists no rectangular grid that contains exactly two million
+rectangles, find the area of the grid with the nearest solution.
+"""
+
+
+from math import ceil, floor, sqrt
+
+
+def solution(target: int = 2000000) -> int:
+    """
+    Find the area of the grid which contains as close to two million rectangles
+    as possible.
+    >>> solution(2000)
+    72
+    """
+    triangle_numbers = [0]
+    for i in range(1, ceil(sqrt(target * 2) * 1.1)):
+        triangle_numbers.append(triangle_numbers[-1] + i)
+
+    best = 0
+    area = 0
+
+    for a, tri_a in enumerate(triangle_numbers[1:], 1):
```

Comment:
Descriptive variable names would be awesome in the long run!
No idea what is `a` or `tri_a`. What does `hit` even mean? What is `best`?

Reply:
Good point, fixed.